cabal-install/cabal-install.cabal

```diff
@@ -333,7 +333,6 @@ executable cabal
         zlib       >= 0.5.3    && < 0.7,
         hackage-security >= 0.5.2.2 && < 0.6,
         text       >= 1.2.3    && < 1.3,
-        zip-archive >= 0.3.2.5 && < 0.4,
```

Comment:
Modify `.pp` instead!

Reply:
That was fast! Will do @ both your comments